packages/insomnia-importers/src/cli.ts

```diff
@@ -32,17 +26,15 @@ module.exports.go = async function() {
   // ~~~~~~~~~~~~~~~~~~~~~~ //
   // Convert the input file //
   // ~~~~~~~~~~~~~~~~~~~~~~ //
-
   const fullInputPath = path.resolve(inputPath);
   const fileContents = fs.readFileSync(fullInputPath, 'utf8');
-
-  const result = await importers.convert(fileContents);
+  const result = await convert(fileContents);
   const exportContents = JSON.stringify(result.data, null, 2);
 
   // ~~~~~~~~~~~~~~~~ //
   // Write the output //
   // ~~~~~~~~~~~~~~~~ //
-
+  const outputPath = program.args[1];
```

Comment:
it does look that way, but `program.output` does not exist (if it ever even did).  I was alerted of this by typescript 
![Screenshot_20210405_152702](https://user-images.githubusercontent.com/15232461/113616648-60fac100-9623-11eb-97cb-7221a6fd2897.png)

But if you investigate by adding
![Screenshot_20210405_152742](https://user-images.githubusercontent.com/15232461/113616720-7839ae80-9623-11eb-8936-ebbf562eec21.png)

you get the following output (which confirms that `program.output` does not exist):
```console
dimitri@dimitri:~/src/github.com/Kong/insomnia$ npm run workspace insomnia-importers build

> insomnia@1.0.0 workspace /home/dimitri/src/github.com/Kong/insomnia
> $(npm bin)/lerna run --scope "insomnia-importers" "build"

lerna notice cli v3.22.1
lerna notice filter including "insomnia-importers"
lerna info filter [ 'insomnia-importers' ]
lerna info Executing command in 1 package: "npm run build"
lerna info run Ran npm script 'build' in 'insomnia-importers' in 1.5s:

> insomnia-importers@2.2.29 build /home/dimitri/src/github.com/Kong/insomnia/packages/insomnia-importers
> tsc --build tsconfig.build.json

lerna success run Ran npm script 'build' in 1 package in 1.5s:
lerna success - insomnia-importers
dimitri@dimitri:~/src/github.com/Kong/insomnia$ ./packages/insomnia-importers/bin/cli
["Command","CommanderError","Help","InvalidOptionArgumentError","Option","_actionHandler","_actionResults","_addImplicitHelpCommand","_aliases","_allowExcessArguments","_allowUnknownOption","_args","_argsDescription","_combineFlagAndOptionalValue","_defaultCommandName","_description","_enablePositionalOptions","_events","_eventsCount","_executableFile","_executableHandler","_exitCallback","_hasHelpOption","_helpCommandDescription","_helpCommandName","_helpCommandnameAndArgs","_helpConfiguration","_helpDescription","_helpFlags","_helpLongFlag","_helpShortFlag","_hidden","_maxListeners","_name","_optionValues","_outputConfiguration","_passThroughOptions","_scriptPath","_storeOptionsAsProperties","_usage","_version","_versionOptionName","args","commands","options","parent","program","rawArgs"]
undefined
Input path not specified
```

Reply:
Interesting! Well then 👍

Reply:
In earlier versions of Commander the options would be available as properties on the program object itself. They changed that API over time (I remember when working through inso) but because this is such an old implementation my guess is that's why output is being accessed directly on the `program` object.

Reply:
thanks for the background :)

That actually vaguely sounds familiar now that you say it.